src/herder/HerderImpl.cpp

```diff
@@ -892,6 +892,27 @@ HerderImpl::getCurrentLedgerSeq() const
     return res;
 }
 
+uint32
+HerderImpl::getLowNeedsSCPLedgerSeq() const
+{
+    // computes the smallest ledger for which we *think* we need more SCP
+    // messages
+    // we ask for messages older than the latest ledger in case they have SCP
```

Comment:
can we replace "latest ledger" with "lcl" for clarity?

Reply:
done